contracts/Account/EIP712SignerRecovery.sol

```diff
@@ -32,6 +32,6 @@ contract EIP712SignerRecovery {
     ));
 
     // recover the signer address from the signed messageHash and return
-    signer = ECDSA.recover(messageHash, signature);
+    return ECDSA.recover(messageHash, signature);
```

Comment:
@Kyrrui if we're doing it this way, the function definition should be `returns (address)` so it's not defining an unused `signer` variable